packages/components/src/popover/index.js

```diff
@@ -400,6 +400,7 @@ const Popover = (
 			};
 			updateFrameOffset();
 			defaultView.addEventListener( 'resize', updateFrameOffset );
+			update();
```

Comment:
Added this for good measure, but I'm not completely sure it's needed.

Reply:
Shouldn't the call to `update` happen inside the `updateFrameOffset`, if anything?

Reply:
I thought the same with Marco here..

Reply:
This seems to fix:

>While the block toolbar is showing for the selected block, opening the "W" menu causes the toolbar to get out of sync with its anchor. The toolbar repositions itsef correctly only after "forcing" an update (e.g by scrolling)

Reply:
I can confirm that the change that @ntsekouras pushed (moving `update()` inside the `updateFrameOffset` function) fixes that issue

Reply:
That's where my brain intended it to be, but my hands obviously did something different 😄 

Thanks for fixing it.